composer.json

```diff
@@ -13,12 +13,12 @@
         "doctrine/event-manager": "^2.0",
         "editorconfig-checker/editorconfig-checker": "^10.6.0",
         "ergebnis/composer-normalize": "^2.43.0",
-        "phpstan/phpstan": "^1.11.5",
+        "phpstan/phpstan": "^1.11.8",
         "phpstan/phpstan-phpunit": "^1.4.0",
         "phpstan/phpstan-strict-rules": "^1.6.0",
-        "phpunit/phpunit": "^10.5.24",
-        "shipmonk/composer-dependency-analyser": "^1.5.3",
-        "shipmonk/phpstan-rules": "^3.0",
+        "phpunit/phpunit": "^11.2.9",
```

Comment:
Looks like our "dont update major versions" ignore is not working: https://github.com/shipmonk-rnd/doctrine-hint-driven-sql-walker/blob/master/.github/dependabot.yml#L14